octoprint_findmymrbeam/__init__.py

```diff
@@ -318,7 +323,7 @@ def is_enabled(self):
 	def _not_disabled(self):
 		enabled = False
 		try:
-			if self._support_mode:
+			if len(self._get_internal_modes()) > 0:
```

Comment:
`len` is unnecessary. could just be `if self._get_internal_modes():` as empty lists are False-ish

Reply:
yes. and I wanted to make it explicit since it's not so obvious here that the return val is a list